### ISSUE TITLE
Setting height of button

### ISSUE THREAD

**Issue by rasmusandersson5** [google-pay/compose-pay-button#4]
Is there any way of setting the height of the button? `Modifier.height` only sets the height of the container and not the actual button

**Comment by dmengelt** [comment]
@rasmusandersson5 Right. This is currently not possible. Could you elaborate a bit more on your use case? Maybe share a screenshot of your screen indicating why you need the height to be adjustable? Thanks for your help.

**Comment by rasmusandersson5** [comment]
Hi!

Basically I want the button to align with other button heights from our design system.
Show a Google Pay button together with, let's say, our own "Continue" button looks a bit bad now since the height is not exactly the same.

**Comment by dmengelt** [comment]
Thanks for the clarifications @rasmusandersson5 
Any chance you could share a screenshot with me? I would love to see how it looks...

**Comment by rasmusandersson5** [comment]
This is probably all I can give you atm 😁 

<img width="448" alt="Screenshot 2023-11-05 at 14 32 55" src="https://github.com/google-pay/compose-pay-button/assets/78341219/098098e4-4fac-43df-8eea-98c48e3fa598">

**Comment by cmargonis** [comment]
Hello, posting here to add that this is something that we'd also like (same scenario 😄 )

**Comment by naltynbekkz** [comment]
+1

**Comment by LaurentLixfe** [comment]
Hello, same issue for me. According to the guidelines, each payment button should have the same size, unfortunately that's not possible without changing the height

**Comment by jmartinVMLYR** [comment]
Same ask from me. We show the GPay button adjacent to buttons for other payment options, and we want to be able to bring some uniformity to the buttons without having to restyle our other buttons to match the GPay button

**Comment by sofi-kry** [comment]
Same here, any updates?
